ibtk/src/lagrangian/FEProjector.cpp

```diff
@@ -317,34 +320,43 @@ FEProjector::buildLumpedL2ProjectionSolver(const std::string& system_name)
         for (MeshBase::const_element_iterator el_it = el_begin; el_it != el_end; ++el_it)
         {
             const Elem* const elem = *el_it;
+#if NDEBUG && LIBMESH_VERSION_LESS_THAN(1, 5, 0)
```

Comment:
I think you mixed up `NDEBUG` and `DEBUG` here.

I am always in favor of running these sorts of checks - anything thats not in an inner loop is negligibly cheap.

Reply:
Fixed.